### DISCUSSION TITLE
Clearify “Zigbee channel utilization” warnings reported in Home Assistant logs?

### DISCUSSION THREAD

**Discussion by Hedda** [zigpy/zigpy#1209] [ZHA development]
FYI, ZHA users are reporting high percentage of “Zigbee channel xx utilization” immediately after restarting Home Assistant 2023.4 and later:

https://www.reddit.com/r/homeassistant/comments/12dnqri/zha_channel_utilization_warning_can_i_run_a_scan/

https://community.home-assistant.io/t/zigbee-channel-15-utilization-is-97-04-in-logs-immediately-after-restart/559094

https://community.home-assistant.io/t/zigbee-channel-11-utilization-is-92-06/558681/

Suggest might be a good idea to add a clarifying comment to that message in the log to explain why a user can see high percentages there?

Secondly, it could perhaps be a good idea to also offer the option to "Perform energy scan" on all Zigbee channels as a feature in ZHA UI?

https://github.com/zigpy/zigpy-cli#performing-an-energy-scan

**Comment by MattWestb** [comment]
"online energy scan" is not allowed then one network is formed then its disabling broadcast and routing in Zigbee 3 and is not use doing it online better doing it off line with one other radio.

**Comment by puddly** [comment]
This is expected, the message was introduced in 2023.4.0.

> Suggest might be a good idea to add a clarifying comment to that message in the log to explain why a user can see high percentages there?

There is one:
<img width="765" alt="image" src="https://github.com/zigpy/zigpy/assets/32534428/b2b63c19-a800-4b35-a28d-afdea29a8cad">

> Secondly, it could perhaps be a good idea to also offer the option to "Perform energy scan" on all Zigbee channels as a feature in ZHA UI?

It's too low-level of a feature and is already present in the ZHA diagnostics JSON. The main purpose of energy scanning is to find a new channel, and the `auto` channel selection algorithm for channel migration will do just that.
